grunt-configs/shell.js

```diff
@@ -24,13 +24,31 @@ module.exports = function(grunt, options) {
                      'static/abtests.json'
         },
 
-        jspmFaciaTool: {
+        jspmInstallFaciaTool: {
             command: 'node ../../node_modules/jspm/jspm.js install',
             options: {
                 execOptions: {
                     cwd: 'facia-tool/public'
                 }
             }
+        },
+
+        jspmInstallStatic: {
+            command: './node_modules/.bin/jspm install',
```

Comment:
Frontend does it regularly already (for facia tool). I didn't configure any credentials on TC, it magically works... But it should be sorted out.
Btw it works better after I moved raven from github to npm...